tests/PermissionedControllerEmissionTestMATICXPolygon.t.sol

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Polygon, AaveV3PolygonAssets} from 'aave-address-book/AaveV3Polygon.sol';
+import {IEmissionManager, ITransferStrategyBase, RewardsDataTypes, IEACAggregatorProxy} from '../src/interfaces/IEmissionManager.sol';
+import {LMSetupBaseTest} from './utils/LMSetupBaseTest.sol';
+import {IPermissionedPayloadsController, PayloadsControllerUtils, IPayloadsControllerCore} from 'aave-address-book/governance-v3/IPermissionedPayloadsController.sol';
+
+// TEMPORARY IMPORTS
+import {IOwnable} from 'solidity-utils/contracts/transparent-proxy/interfaces/IOwnable.sol';
+import {TransparentProxyFactory} from 'solidity-utils/contracts/transparent-proxy/TransparentProxyFactory.sol';
+import {Executor} from 'aave-governance-v3/contracts/payloads/Executor.sol';
+import {PermissionedPayloadsController} from 'aave-governance-v3/contracts/payloads/PermissionedPayloadsController.sol';
+import {IERC20} from 'forge-std/interfaces/IERC20.sol';
+
+contract PermissionedControllerEmissionTestMATICXPolygon is LMSetupBaseTest {
```

Comment:
I would wrap this file in a folder along with its script, so its similar to the structure generated by generator